c7n/resources/dms.py

```diff
@@ -55,6 +55,10 @@ class resource_type(TypeInfo):
         enum_spec = ('describe_endpoints', 'Endpoints', None)
         arn = id = 'EndpointArn'
         name = 'EndpointIdentifier'
+        arn_type = 'endpoint'
+        universal_taggable = True
```

Comment:
this should be `universal_taggable = object()` else you'll get compatibility aliases.

Reply:
Updated in 848c9a0 thanks!